book/creating-a-basic-website/adding-localizations.rst

```diff
@@ -56,3 +56,39 @@ the content. In case there is a ghost page - which means that the page is not
 translated into the current localization - this tree will be used to determine
 the "closest" language available.
 
+Localizing Strings
```

Comment:
That's what I also said... I think explaining pure symfony features in our docs doesn't make too much sense.